validator-api/src/network_monitor/monitor/sender.rs

```diff
@@ -73,8 +72,7 @@ struct FreshGatewayClientData {
     // SECURITY:
     // since currently we have no double spending protection, just to get things running
```

Comment:
Well, now we DO have double spending protection, no?

Reply:
Just for the non-coconut bandwidth afaik. I'll update the comment.